docs/api-changes.md

```diff
@@ -0,0 +1,22 @@
+API Changes
```

Comment:
Thanks @thockin - sounds good to me. I will merge those two files (most probably tomorrow).